File: PRACTICE/avgProdLarg.js
const prompt = require("prompt-sync")();

    numOne = Number(prompt("Enter first number"));
    numTwo = Number(prompt("Enter second number"));
    numThree = Number(prompt("Enter third number"))

    squareOne = numOne * numOne;
    squareTwo = numTwo * numTwo;
    squareThree = numThree * numThree;

    average = (numOne + numTwo + numThree)/3;
    product = (numTwo * numTwo * numTwo);
    
    largest = numOne;
    if(numTwo > numOne && numTwo > numThree){
    largest = numTwo;
    }
    else if(numThree > numOne && numThree > numOne){
    largest = numThree;
    }
    else{
    console.log("Error")
    }

    console.log("average is: " + average);
    console.log("product is: " + product);
    console.log("largest is: "+ largest);

File: PRACTICE/avg_prod_larg.py
num_one = int(input("Enter first number "))
num_two= int(input("Enter second number "))
num_three= int(input("Enter third number"))

average = (num_one + num_two + num_three)/3

product = num_one * num_two * num_three


largest = num_one
if(num_two > num_one and num_two > num_three):
    largest = num_two    
    
elif(num_three > num_one and num_three > num_two):
    largest = num_three
    
else:
    print("Error")
    

print("Avergae of numbers is: ", average)
print("Product of numbers is: ", product)
print("Largest of numbers is: ", largest)
